SqlServer.Dac/DataTypeView.cs

```diff
@@ -34,7 +35,7 @@ public DataTypeView(string name, DataTypeReference dataType, DataTypeViewType ty
             Name = name;
             DataType = dataType.Name.Identifiers.Last().Value;
             var length = (dataType as ParameterizedDataTypeReference)?.Parameters.FirstOrDefault()?.Value;
-            Length = length?.ToLower() == "max" ? -1 : Convert.ToInt32(length);
+            Length = length?.ToUpperInvariant() == "max" ? -1 : Convert.ToInt32(length, CultureInfo.InvariantCulture);
```

Comment:
`ToLowerInvariant` or change `max` to `MAX`

Reply:
@Saibamen Well spotted!